site/docs/encoding.md

```diff
@@ -236,7 +236,7 @@ Similar to [`detail`](#detail), you can use an array of field definitions. Vega-
 {:#href}
 ## Hyperlink Channel
 
-By setting the `href` channel, a mark becomes a hyperlink. The specified URL is loaded upon a mouse click. When the `href` channel is specified, the [`cursor` mark property](mark.html#hyperlink) is set to `pointer` to serve as affordance for hyperlinks.
```

Comment:
This should be `"pointer"`

Reply:
Is it not "pointer" in the commit? Do I need a certain kind of quotation mark?

Reply:
Use \` and "

Reply:
Got it